src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 0.3s;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-main-accent;
+    margin-bottom: 4px;
+    transition: color 0.3s;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-secondary;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+
+    &-reviews {
+      display: flex;
+      align-items: center;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $color-main-accent;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-label {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $color-secondary;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $color-main-accent;
+    }
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $color-link-active;
+    border-radius: 5px;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: #fff;
+    text-decoration: none;
+    border: 1px solid $color-link-active;
+    padding: 11px 0;
+    transition: all 0.3s;
+
+    &:hover {
+      color: $color-link-active;
+      background-color: #fff;
+      transition: all 0.3s;
```

Comment:
You dont need to set transition on hover as it is already set on the origin element
```suggestion
```